co-log-core/co-log-core.cabal

```diff
@@ -28,7 +28,7 @@ library
                            Colog.Core.Class
                            Colog.Core.Severity
 
-  build-depends:       base >= 4.11 && < 5
+  build-depends:       base >= 4.10 && < 5
```

Comment:
The point to make it `< 4.13` is that when the new `base` version will be released we still can guarantee that this version of our library will work as expected. If we specify `< 5` this would mean that the breaking changes of `base-4.13.*` or later can break this particular version of our library and we will need to make a revision on Hackage to tighten the `base` bounds to `< 4.13`. I hope this explanation makes sense.

Reply:
Yes, that makes sense 👍